modules/internal/String.chpl

```diff
@@ -1406,6 +1462,9 @@ module String {
   //
   // Append
   //
+  /*
+     Appends the string `lhs` with the string `rhs`.
```

Comment:
I'd prefer this written as "Appends the string 'rhs' to the string 'lhs'." as the action is more direct then.